### PR TITLE
Add migration for uniques (distincts)

### PR DESCRIPTION
**Pull request by joshuatobin**
@reidmix @lexelby can you give me some advice on how the best way would be to add a unique constraint on responders?

### CODE REVIEW

**Comment by reidmix** [comment]
Add a scope to the association `-> { uniq }` -- you may be able to remove the checks `unless responders...` 

https://stackoverflow.com/questions/16569994/deprecation-warning-when-using-has-many-through-uniq-in-rails-4

File: app/models/incident.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Incident < ActiveRecord::Base
   has_many :remediations, through: :retrospectives
   belongs_to :category
   has_many :timeline_entries, -> { order(timestamp: :asc) }
-  has_and_belongs_to_many :responders, join_table: :incidents_responders, class_name: "User"
+  has_and_belongs_to_many :responders, -> { distinct }, join_table: :incidents_responders, class_name: "User"
```
Comment (joshuatobin) [inline app/models/incident.rb]: `uniq` changed in rails 5.1 -> https://github.com/rails/rails/commit/adfab2dcf4003ca564d78d4425566dd2d9cd8b4f

**Comment by joshuatobin** [comment]
Verified adding myself multiple times: 
screen shot 2016-10-25 at 10 16 47 am

File: app/models/incident.rb (hunk 1 above)
Comment (reidmix) [inline app/models/incident.rb]: oh interesting -- looks like they're trying to match sql syntax.

**Comment by reidmix** [comment]
LGTM 👍

**Pull request merged**